code/game/turfs/turf.dm

```diff
@@ -21,10 +21,10 @@
 
 	var/explosion_level = 0	//for preventing explosion dodging
 	var/explosion_id = 0
-	
+
 	var/list/decals
 
-/turf/New()
+/turf/Initialize()
 	..()
 
 	levelupdate()
```

Comment:
Ya, I just confirmed that the smoothing of the 1st two zlevels as a seperate step in SSicon_smooth's initialize() is no longer needed (potentially because of this pr since i haven't bothered to test on master).

Also, changing queue_smooth()'s `SSicon_smooth.smooth_queue[A] = A` to `SSicon_smooth.smooth_queue += A` and tracking it by adding a bit flag to `atom.smooth` to avoid dupes removes all of the overhead of queue_smooth, going down from 5 seconds to 0.2.

I'll just pr that last bit as a pr